clr/main.py

```diff
@@ -1,6 +1,21 @@
 import sys
+import os
+import getpass
+import beeline
 from clr.commands import resolve_command, get_namespace
+from clrenv import env
 
+def init_beeline(service_name):
+    if env.honeycomb.writekey is not None:
```

Comment:
Would that also be checking to make sure writekey is there and has a value?

Reply:
it does not, we could certainly check for both. As is without a honeycomb entry at the base level of the config it will fail (as you noticed trying to run unit tests).

My thinking is if honeycomb is missing from from the config silently disable all honeycomb usage, but if it is present it is reasonable to error out when the writekey is missing (they tried but failed to enable it)

Reply:
done